src/components/CodeBlock/CodeBlock.tsx

```diff
@@ -14,15 +14,15 @@ export const CodeBlock = ({ ref, source, ...props }: CodeBlockProps) => {
         >
```

Comment:
Should it have `language="json"` by default?

Reply:
@rosinghal

Reply:
I think it shouldn't because it's always to specify which language are you expecting the output.